doc/src/modules/physics/control/control_problems.rst

```diff
@@ -0,0 +1,341 @@
+    >>> # Series equivalent, considering (Input)→[C]→[P]→(Output).
+    >>> pprint(P*C)
+    [1    2  ]    [1  1]
+    [-  -----]    [-  -]
+    [s  s + 2]    [1  1]
+    [        ]   *[    ]
+    [0    3  ]    [2  2]
+    [-    -  ]    [-  -]
+    [1    1  ]{t} [1  1]{t}
+    >>> pprint((C*P).doit())
+    [1  3*s + 8 ]
+    [-  ------- ]
+    [s   s + 2  ]
+    [           ]
+    [2  6*s + 16]
+    [-  --------]
+    [s   s + 2  ]{t}
+    >>> pprint((P*C).doit())
+    [ 5*s + 2    5*s + 2 ]
+    [---------  ---------]
+    [s*(s + 2)  s*(s + 2)]
+    [                    ]
+    [    6          6    ]
+    [    -          -    ]
+    [    1          1    ]{t}
+
+    Subpart 2
+
+    >>> tfm_feedback = MIMOFeedback(P, C, sign=-1)
+    >>> pprint(tfm_feedback.doit())  # ((I + P*C)**-1)*P
+    [    7*s + 14          -s - 6    ]
+    [---------------  ---------------]
+    [   2                2           ]
+    [7*s  + 19*s + 2  7*s  + 19*s + 2]
+    [                                ]
+    [                     2          ]
+    [   -6*s - 12      3*s  + 9*s + 6]
+    [---------------  ---------------]
+    [   2                2           ]
+    [7*s  + 19*s + 2  7*s  + 19*s + 2]{t}
+
+
+
+Example 5
+---------
+
+        .. image:: Q5.svg
+
+Given,
+
+        .. math::
+            G1 &= \frac{1}{10 + s}\\\\
+
+            G2 &= \frac{1}{1 + s}\\\\
+
+            G3 &= \frac{1 + s^2}{4 + 4s + s^2}\\\\
+
+            G4 &= \frac{1 + s}{6 + s}\\\\
+
+            H1 &= \frac{1 + s}{2 + s}\\\\
+
+            H2 &= \frac{2 \cdot (6 + s)}{1 + s}\\\\
+
+            H3 &= 1\\
+
+Where $s$ is the variable of the transfer function (in Laplace Domain).
+
+Find -
+
+1. The equivalent Transfer Function representing the system given above.
+2. Pole-Zero plot of the system.
+
+
+Solution-
+
+    >>> from sympy.abc import s
+    >>> from sympy.physics.control import *
+    >>> G1 = TransferFunction(1, 10 + s, s)
+    >>> G2 = TransferFunction(1, 1 + s, s)
+    >>> G3 = TransferFunction(1 + s**2, 4 + 4*s + s**2, s)
+    >>> G4 = TransferFunction(1 + s, 6 + s, s)
+    >>> H1 = TransferFunction(1 + s, 2 + s, s)
+    >>> H2 = TransferFunction(2*(6 + s), 1 + s, s)
+    >>> H3 = TransferFunction(1, 1, s)
+    >>> sys1 = Series(G3, G4)
+    >>> sys2 = Feedback(sys1, H1, 1).doit()
+    >>> sys3 = Series(G2, sys2)
+    >>> sys4 = Feedback(sys3, H2).doit()
+    >>> sys5 = Series(G1, sys4)
+    >>> sys6 = Feedback(sys5, H3)
+    >>> sys6  # Final unevaluated Feedback object
+    Feedback(Series(TransferFunction(1, s + 10, s), TransferFunction((s + 1)**3*(s + 2)*(s + 6)**2*(s**2 + 1)*(-(s + 1)**2*(s**2 + 1) + (s + 2)*(s + 6)*(s**2 + 4*s + 4))*(s**2 + 4*s + 4)**2, (s + 1)*(s + 6)*(-(s + 1)**2*(s**2 + 1) + (s + 2)*(s + 6)*(s**2 + 4*s + 4))*((s + 1)**2*(s + 6)*(-(s + 1)**2*(s**2 + 1) + (s + 2)*(s + 6)*(s**2 + 4*s + 4))*(s**2 + 4*s + 4) + (s + 1)*(s + 2)*(s + 6)*(2*s + 12)*(s**2 + 1)*(s**2 + 4*s + 4))*(s**2 + 4*s + 4), s)), TransferFunction(1, 1, s), -1)
```

Comment:
Seeing this form makes me think that displaying an ascii block diagram would be more useful to the user. This is like seeing the srepr of a sympy expression.

Reply:
My apologies, but I did not get what you are suggesting. Can you expand or maybe show through an example ... wat you expect here ?

Reply:
You don't need to fix anything for this, but I was just reflecting on how we may want a better repr for these expressions. One that is easily readable.

Reply:
I will keep this comment in mind for any future discussions on possible improvements.

Reply:
I agree with Jason. This looks way too confusing, even to people who might know what's going on here. 
But it is out of scope for this PR. We can discuss this in Phase 2 and see if we can come up with something better

Reply:
Sure. I have saved this comment for phase 2 discussions.